cadence/nfts/blind-edition-nft/README.md

```diff
@@ -1,3 +1,89 @@
 # Blind Edition NFT Template
 
-TODO: add documentation
+The Blind Edition NFT template is similar to the blind NFT contract, but for edition-based NFTs.
+
+With this contract, edition-based NFTs are initially minted in "blind" form with their metadata visible
+but their serial numbers hidden.
+
+The minter can then reveal serial numbers by publishing them to the blockchain at a later date.
+
+Using a cryptographic hash, NFT owners can verify that their serial number matches the original blind NFT.
+
+## How blind minting works
+
+We use the same technique as we do for standard blind NFTs, except that the we
+only hash the serial number rather than the entire metadata object.
+
+For edition-based NFTs, all the metadata values are known at mint time (except the serial number).
+We don't gain much from hiding the metadata because it would be effectively revealed for all NFTs in an edition
+as soon as one NFT is revealed, due to the fact that all NFTs in an edition share the same metadata.
+
+### Generating the metadata hash
+
+For edition-based NFTs, we hash the serial number combined with a random salt.
+
+Here's some pseudocode:
+
+```
+metadataHash = sha3(
+  SALT,
+  encodeInt(serialNumber)
+)
+```
+
+where `SALT` is a random 32 byte sequence.
+
+### Revealing the metadata
+
+To reveal the metadata for an NFT, we publish the salt and serial number to the NFT smart contract
+using the [reveal.cdc](./transactions/reveal.template.cdc) transaction.
+
+The contract then hashes the salt and serial number for each NFT directly on-chain using Cadence's crypto standard library.
+It will reject the serial number if it does not hash to the same value that was set at mint time.
+
+## Contract generation
+
+The template requires the following inputs when generating a contract:
+
+- `contractName` - the name of the contract.
+- `fields` - the metadata fields to include on the contract.
+- `views` - the metadata views to include on the contract.
+- `saveAdminResourceToContractAccount` - a special boolean flag that indicates whether the `Admin` resource should be saved to the contract account at mint time. If false, the deployer must pass to the contract initializer an `AuthAccount` that will receive the `Admin` resource.
+
+## TypeScript wrapper
+
+This template has an accompanying TypeScript wrapper in `@freshmint/core` implemented in [`BlindEditionNFTContract.ts`](../../../packages/core/contracts/BlindEditionNFTContract.ts).
+
+The wrapper contains functions to generate a contract, deploy it and mint NFTs.
```

Comment:
Add: "See nodejs.md documentation's [Blind Edition NFTs section](freshmint/tree/main/docs/nodejs.md#blind-edition-nfts) for more details."